optionals/compat_rhs_usf3/script_component.hpp

```diff
@@ -1,4 +1,4 @@
-#define COMPONENT rhsusf_c_weapons_comp
+#define COMPONENT compat_rhs_usf3
```

Comment:
Note: `COMPONENT` was not the same as the PBO prefix, thus `PREP` did not work. Since this compatibility addon does not only handle weapons I changed it to the PBO prefix.